website/addons/owncloud/tests/test_views.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+from nose.tools import *  # noqa
+import mock
+
+import httplib as http
+from tests.factories import AuthUserFactory
+
+from framework.auth.decorators import Auth
+
+from website.util import api_url_for
+from website.addons.base.testing import views
+from website.addons.owncloud.model import OwnCloudProvider
+from website.addons.owncloud.serializer import OwnCloudSerializer
+from website.addons.owncloud.tests.utils import (
+    create_mock_owncloud, OwnCloudAddonTestCase,create_external_account
+)
+
+class TestAuthViews(OwnCloudAddonTestCase):
+
+    def test_deauthorize(self):
```

Comment:
This test doesn't appear to be doing much of anything that the test of the same name in ownCloud's `TestNodeSettings` isn't doing, aside from sending it through flask. Consider removing